packaging/src/docker/6.x/docker-compose.yml

```diff
@@ -1,56 +1,82 @@
-version: '3'
+# This docker-compose file will spin up an ACS cluster on a local host or on a server and it requires a minimum of 12GB Memory to distribute among containers.
+# Limit container memory and assign X percentage to JVM.  There are couple of ways to allocate JVM Memory for ACS Containers
+# For example: 'JAVA_OPTS: "$JAVA_OPTS -XX:+PrintFlagsFinal -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap"'
+# But, as per Oracle docs (https://docs.oracle.com/javase/9/gctuning/parallel-collector1.htm#JSGCT-GUID-CAB83393-3438-44ED-98F0-D15641B43C7D)
+# If container memory is not explicitly set, then the above flags will default max heap to 1/4th of container's memory which may not be ideal.
+# Hence, setting up explicit Container memory and then assigning a percentage of it to the JVM for performance tuning.
+
+# Using version 2 as 3 does not support resource constraint options (cpu_*, mem_* limits) for non swarm mode in Compose
+version: "2"
+
 services:
-  alfresco:
-    image: alfresco/alfresco-content-repository${ALFRESCO_EDITION}:${ALFRESCO_TAG}
-    environment:
-      JAVA_OPTS : "
-        -Ddb.driver=org.postgresql.Driver
-        -Ddb.username=alfresco
-        -Ddb.password=alfresco
-        -Ddb.url=jdbc:postgresql://postgres:5432/alfresco
-        -Dsolr.host=search
-        -Dsolr.port=8983
-        -Dsolr.secureComms=none
-        -Dsolr.base.url=/solr
-        -Dindex.subsystem.name=solr6
-        -Dmessaging.broker.url=\"failover:(nio://activemq:61616)?timeout=3000&jms.useCompression=true\"
-        "
-    ports:
-      - "7203:7203" #JMX connect via service:jmx:rmi:///jndi/rmi://localhost:7203/jmxrmi
-      - "5005:5005" #Java debugging
-      - "8081:8080" #Browser port for Alfresco
-  share:
-    image: alfresco/alfresco-share:${SHARE_TAG}
-    environment:
-      - REPO_HOST=alfresco
-      - REPO_PORT=8080
-    ports:
-      - 8082:8080 #Browser port for Share
-  postgres:
-    image: postgres:${POSTGRES_TAG}
-    environment:
-      - POSTGRES_PASSWORD=alfresco
-      - POSTGRES_USER=alfresco
-      - POSTGRES_DB=alfresco
-    ports:
-      - 5432:5432
-  search:
-    image: quay.io/alfresco/search-services:${SEARCH_TAG}
-    environment:
-      #Solr needs to know how to register itself with Alfresco
-      - SOLR_ALFRESCO_HOST=alfresco
-      - SOLR_ALFRESCO_PORT=8080
-      #Alfresco needs to know how to call solr
-      - SOLR_SOLR_HOST=search
-      - SOLR_SOLR_PORT=8983
-      #Create the default alfresco and archive cores
-      - SOLR_CREATE_ALFRESCO_DEFAULTS=alfresco,archive
-    ports:
-      - 8083:8983 #Browser port
-  activemq:
-    image: alfresco/alfresco-activemq:5.15.6
-    ports:
-      - 8161:8161 # Web Console
-      - 5672:5672 # AMQP
-      - 61616:61616 # OpenWire
-      - 61613:61613 # STOMP
\ No newline at end of file
+    alfresco:
+        image: alfresco/alfresco-content-repository-community:${ALFRESCO_TAG}
```

Comment:
Change to: alfresco/alfresco-content-repository${ALFRESCO_EDITION}:${ALFRESCO_TAG} please
Insight Engine (sql support) is to be run with Enterprise edition.

Reply:
The same from above.